libfs/fs.go

```diff
@@ -526,6 +593,18 @@ func (fs *FS) Stat(filename string) (fi os.FileInfo, err error) {
 		err = translateErr(err)
 	}()
 
+	if fs.empty && (filename == "" || filename == ".") {
+		// TODO: use FileInfoFast.
```

Comment:
What's `FileInfoFast`?

Reply:
That's something in an unmerged PR from @songgao: #1950.  It avoids needing to pass in a node here.

Reply:
Sorry I haven't got it merged yet; will try to get it done today or tomorrow!